packages/eas-cli/src/project/ios/entitlements.ts

```diff
@@ -22,14 +27,39 @@ export async function getManagedApplicationTargetEntitlementsAsync(
       ...process.env,
       ...env,
     };
-    const { exp } = await getPrebuildConfigAsync(projectDir, { platforms: ['ios'] });
-
-    const expWithMods = await compileModsAsync(exp, {
-      projectRoot: projectDir,
-      platforms: ['ios'],
-      introspect: true,
-      ignoreExistingNativeFiles: await hasIgnoredIosProjectAsync(projectDir, vcsClient),
-    });
+
+    let expWithMods: ExportedConfig;
+    try {
+      const { stdout } = await spawnAsync(
+        'npx',
+        ['expo', 'config', '--json', '--type', 'introspect'],
```

Comment:
I think the issue might be that this command now does the config for both platforms (https://github.com/expo/expo/blob/main/packages/%40expo/cli/src/config/configAsync.ts#L66-L75) while previously it only did the `getPrebuildConfigAsync` and `compileModsAsync` for ios.